changelog.mdx

```diff
@@ -4,6 +4,38 @@ description: "All the nice stuff you asked for"
 icon: "sparkles"
```

Comment:
and yes selector editing is relevant: "You can now edit the playwright selectors that are used to find the element in question for the step"

Reply:
ok, I'll add it